recipes-core/packagegroups/packagegroup-application-layer.bb

```diff
@@ -11,4 +11,5 @@ PV = "4.0.0"
 RDEPENDS:packagegroup-application-layer = " \
                                            residentui \
                                            rdkresidentapp \
+                                           ${@bb.utils.contains('DISTRO_FEATURES', 'enable-dab', 'dab-adapter mosquitto', '', d)} \
```

Comment:
Missing package ARCH assignments for DAB Adapter related recipes.
If mosquitto is RDEPENDS; should we really need that to be added explicitely ?